### PR TITLE
TWIS-77/MoreActionsMenuList

### PR DESCRIPTION
**Pull request by Mujayt0231**
# TWIS-77/More Action Menu List

## Link to Story

https://re-boot.atlassian.net/browse/TWIS-77

## Description

More Actions Menu List has been created

## Screenshot(s) / GIF(s):

Screenshot 2024-03-20 at 3 15 41 PM
Screenshot 2024-03-20 at 3 24 39 PM

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Dependency / Configuration update
- [ ] Documentation update

## Checklist:

- [x] My code follows the project's style guidelines
- [x] I have created a Storybook story for my component(s)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Jack-Vu — DISMISSED** [review]
LGTM!

File: src/app/_components/MoreActionsMenu/MoreActionsMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  List,
+  ListItemIcon,
+  ListItemText,
+  ListItemButton,
+} from "@mui/material";
+import MoreHorizIcon from "@mui/icons-material/MoreHoriz";
+import EditIcon from "@mui/icons-material/Edit";
+import DeleteIcon from "@mui/icons-material/Delete";
+
+export type MoreActionsMenuProps = {
```
Comment (Jack-Vu) [inline src/app/_components/MoreActionsMenu/MoreActionsMenu.tsx:17]: Just wondering why export type versus interface here?

**Review by BaburDilshat — DISMISSED** [review]
Looks good!

**Comment by Jack-Vu** [comment]
has conflict might need to update dev and resolve them

**Review by Jack-Vu — APPROVED** [review]
Beautiful code. LGTM!

**Review by Muradil-Erkin — APPROVED** [review]

**Pull request merged**